synapseclient/__main__.py

```diff
@@ -212,7 +212,8 @@ def associate(args, syn):
 
 def copy(args,syn):
     mappings = synapseutils.copy(syn, args.id, args.destinationId, 
-                         copyWikiPage=args.skipCopyWiki, 
+                         copyWikiPage=args.skipCopyWiki,
+                         copyAnnotations=args.skipAnnotations,
                          excludeTypes=args.excludeTypes, 
```

Comment:
copyWiki and copyAnnotations seem to have the opposite meaning to skipCopyWiki.  That is, I would want to set skipCopyWiki=True if I don't want to copy wikis and copyWikiPage=False for the same behavior.